mteb/models/e5_models.py

```diff
@@ -130,6 +130,7 @@
 
 ME5_TRAINING_DATA = {
     **E5_TRAINING_DATA,
+    "XQuAD": ["train"], # trained on SQuAD train dataset
```

Comment:
In mteb it defined as `XQuADRetrieval`
```suggestion
    "XQuADRetrieval": ["train"], # trained on SQuAD train dataset
```